resources/packs/azure/azure.lr.manifest.yaml

```diff
@@ -9,6 +9,56 @@ resources:
     platform:
       name:
       - azure
+  azure.aks.cluster:
+    fields:
+      addonProfiles: {}
+      agentPoolProfiles: {}
+      createdAt: {}
+      dnsPrefix: {}
+      fqdn: {}
+      httpProxyConfig: {}
+      id: {}
+      kubernetesVersion: {}
+      location: {}
+      name: {}
+      networkProfile: {}
+      nodeResourceGroup: {}
+      podIdentityProfile: {}
+      provisioningState: {}
+      rbacEnabled: {}
+      securityProfile: {}
+      storageProfile: {}
+      tags: {}
+      workloadAutoScalerProfile: {}
+    min_mondoo_version: latest
```

Comment:
I'm not sure about this:
Do we need to replace "latest", once we know which version it will be part of?